pkg/reconciler/apiserversource/controller.go

```diff
@@ -36,27 +42,54 @@ const (
 	controllerAgentName = "apiserver-source-controller"
 )
 
+func init() {
+	injection.Default.RegisterController(NewController)
+}
+
+func init() {
+	injection.Default.RegisterClient(withClient)
+}
+
```

Comment:
why two init()?

Reply:
oh thanks! I moved this code to `cfg_host.go`